tests/conftest.py

```diff
@@ -1,80 +1,33 @@
-import pathlib
-import platform
-import pytest
-import shutil
-
 from typing import Dict, Iterator, List
 
-
-@pytest.fixture
-def testing_path() -> pathlib.Path:
-    return pathlib.Path(".") / "tests"
+import pytest
 
 
 @pytest.fixture
 def files_no_CLI() -> Iterator[Dict[str, List[str]]]:
```

Comment:
The type hint is now incorrect. Since the fixture returns a dict, `Iterator` should be removed.